recipes/viralflye/meta.yaml

```diff
@@ -0,0 +1,51 @@
+---
+{% set version = "0.2" %}
+
+package:
+  name: viralflye
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dmitry-Antipov/viralFlye/archive/v{{ version }}.tar.gz
+  sha256: bf7c928628e6fd5c34e6112c40c2fceece62f920acb384838b5e38dd7544af08
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - git
+  host:
+    - python ==3.10.8
+  run:
+    - bcftools ==1.16
```

Comment:
If possible do not pin dependencies that strict. If you do that those packages can hardly be mixed with others in the same environment.

Reply:
Unpinned in https://github.com/bioconda/bioconda-recipes/pull/39987/commits/e2510f0926c1f08529288b23bef9250575a8171a. I only pinned the deps for a reproducible build, but we have the biocontainer for that :)